docs/CHANGELOG.rst

```diff
@@ -5,6 +5,18 @@ Change Log
 All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
+==========
+Unreleased
+==========
+
+Changed
+-------
+- **BACKWARD INCOMPATIBLE:** Use Salmon 1.2.1 in ``salmon-quant`` and
```

Comment:
I would make this into two entries, one for bumping of `salmon-quant` and `salmon-index` and one for bumping processes for strandedness detection.